index.bs

```diff
@@ -942,7 +944,16 @@ spec:css-syntax-3;
 
     1.  Let |p| be [=a new promise=].
 
-    1.  Let |origin| be the [=current settings object=]'s [=environment settings object/origin=].
+    1. [=map/For each=] |optionKey| → <var ignore>optionValue</var> of |options|:
```

Comment:
`|optionKey|` here can be `mediation` or `signal`, which are not credential types.

Perhaps

"For each interface in options’ relevant credential interface objects..."

Like [here](https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/207/8350696...npm1:f138311.html#algorithm-collect-known)

Reply:
Done